mathics/core/symbols.py

```diff
@@ -650,41 +663,97 @@ def is_uncertain_final_definitions(self, definitions) -> bool:
 # PredefineSymbol.
 
 SymbolFalse = PredefinedSymbol("System`False")
+SymbolList = PredefinedSymbol("System`List")
+SymbolTrue = PredefinedSymbol("System`True")
+
+SymbolAbs = Symbol("Abs")
+SymbolDivide = Symbol("Divide")
+SymbolFullForm = Symbol("FullForm")
 SymbolGraphics = Symbol("System`Graphics")
 SymbolGraphics3D = Symbol("System`Graphics3D")
 SymbolHoldForm = Symbol("System`HoldForm")
-SymbolList = PredefinedSymbol("System`List")
+SymbolInputForm = Symbol("InputForm")
 SymbolMachinePrecision = Symbol("MachinePrecision")
 SymbolMakeBoxes = Symbol("System`MakeBoxes")
+SymbolMathMLForm = Symbol("MathMLForm")
 SymbolMaxPrecision = Symbol("$MaxPrecision")
 SymbolMinPrecision = Symbol("$MinPrecision")
 SymbolN = Symbol("System`N")
 SymbolNull = Symbol("System`Null")
 SymbolNumberForm = Symbol("System`NumberForm")
+SymbolOutputForm = Symbol("OutputForm")
+SymbolPlus = Symbol("Plus")
 SymbolPostfix = Symbol("System`Postfix")
+SymbolPower = Symbol("Power")
 SymbolRepeated = Symbol("System`Repeated")
 SymbolRepeatedNull = Symbol("System`RepeatedNull")
 SymbolSequence = Symbol("System`Sequence")
-SymbolTrue = PredefinedSymbol("System`True")
+SymbolStandardForm = Symbol("StandardForm")
+SymbolTeXForm = Symbol("TeXForm")
+SymbolTimes = Symbol("Times")
+SymbolTraditionalForm = Symbol("TraditionalForm")
 
+# NumericOperators uses some of the Symbols above.
+class NumericOperators:
+    """
+    This is a mixin class for Element-like objects that might have numeric values.
+    It adds or "mixes in" numeric functions for these objects like round_to_float().
 
-# The available formats.
+    It also adds methods to the class to facilite building
+    ``Expression``s in the Mathics Python code using Python syntax.
 
-format_symbols = system_symbols(
-    "InputForm",
-    "OutputForm",
-    "StandardForm",
-    "FullForm",
-    "TraditionalForm",
-    "TeXForm",
-    "MathMLForm",
-)
+    So for example, instead of writing in Python:
 
+        to_expression("Abs", -8)
+        Expression(SymbolPlus, Integer1, Integer2)
 
-SymbolInputForm = Symbol("InputForm")
-SymbolOutputForm = Symbol("OutputForm")
-SymbolStandardForm = Symbol("StandardForm")
-SymbolFullForm = Symbol("FullForm")
-SymbolTraditionalForm = Symbol("TraditionalForm")
-SymbolTeXForm = Symbol("TeXForm")
-SymbolMathMLForm = Symbol("MathMLForm")
+    you can instead have:
+        abs(Integer(-8))
+        Integer(1) + Integer(2)
+    """
+
+    def __abs__(self) -> BaseElement:
+        return self.create_expression(SymbolAbs, self)
```

Comment:
I didn't find (at least not in GitHub) where `create_expression` is defined.

Reply:
I had to introduce this weird way to add a method to BaseElement because it was the only way to break the cyclic import. I hope that before the next release we are able to get rid of it.

Reply:
Yes, sure. What I tried to mean is that a more or less short term goal would be to finish disentangling the modules that used to be a single very large file. I guess we are not too far to reach that point, but it is fine if we have a 4.1 release when the evaluation refactory be done